VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -219,7 +219,12 @@ private void bindViewHolderToResponseRow(DiscussionResponseViewHolder holder, fi
         holder.addCommentLayout.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                router.showCourseDiscussionComments(context, comment);
+                if (0 == comment.getChildren().size()) {
```

Comment:
@bguertin: Yes, it would be an API bug, but it's best to implement it so that the client can handle all cases without crashing. @aleffert always tried to get us to "null-proof" the app. Still, this is a larger issue, and can be addressed in another pull request.